gh-123165: make `dis` functions render positions on demand (#123168)
python/cpython: Doc/whatsnew/3.14.rst

  (Contributed by Bénédikt Tran in :gh:`121141`.)

dis
---

> * Added support for rendering :class:`instruction positions <dis.Positions>`
>   when available. This feature is added to the following interfaces via

Reviewer: ```suggestion
* Added support for rendering full source location information of instructions,
 rather than only the line number. This feature is added to the following interfaces via
```

>   when available. This feature is available on the following interfaces via

Reviewer: ```suggestion
  when available. This feature was added to the following interfaces via
```

  (Contributed by Bénédikt Tran in :gh:`121141`.)

dis
---

* Added support for rendering full source location information of
  :class:`instructions <dis.Instruction>`, rather than only the line number.
  This feature is added to the following interfaces via the ``show_positions``

>   flag:

Reviewer: ```suggestion
  argument:
```
